core/src/test/java/com/scalar/db/transaction/consensuscommit/TwoPhaseConsensusCommitTest.java

```diff
@@ -335,4 +336,29 @@ public void rollback_CalledAfterCommitFails_ShouldNeverAbortStateAndRollbackReco
     verify(commit, never()).abortState(snapshot.getId());
     verify(commit, never()).rollbackRecords(snapshot);
   }
+
+  @Test
+  public void
+      rollback_UnknownTransactionStatusExceptionThrownByAbortState_ShouldThrowRollbackException()
+          throws UnknownTransactionStatusException, PreparationException {
+    // Arrange
+    transaction.prepare();
+    when(crud.getSnapshot()).thenReturn(snapshot);
+    when(commit.abortState(snapshot.getId())).thenThrow(UnknownTransactionStatusException.class);
+
+    // Act Assert
```

Comment:
Is it possible to verify `commit.rollbackRecords()` wasn't called?

Reply:
Add assertions for that in 806e19514b80c5b8de764a1a8841d482571a1fa3. Thanks.